VersionControlUI/src/DiffManager.cpp

```diff
@@ -813,6 +843,16 @@ QList<DiffFrame*> DiffManager::createDiffFrames(DiffSetup& diffSetup,
 		auto newNode = const_cast<Model::Node*>(diffSetup.newVersionManager_->nodeIdMap().node(id));
 
 		auto diffNode = new DiffFrame{oldNode, newNode};
+
+		for (auto changeWithNode : changesWithNodes)
```

Comment:
This is used to decide which changed nodes should be stored in the lists of `DiffFrame`. It knows now which changed nodes, in which version it contains.